05-read-write-plot.Rmd

```diff
@@ -421,22 +419,22 @@ saveWidget(mapview_obj, file="my_interactive_map.html")
 
 1. List and describe three types of vector, raster, and geodatabase formats.
 
-1. Name at least differences between `read_sf()` and the more well-known function `st_read()`.
+2. Name at least two differences between `read_sf()` and the more well-known function `st_read()`.
```

Comment:
It does not matter that they are always 1s - Markdown knows what to do: https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#lists